packages/fiori-freestyle-writer/templates/common/add/ui5-local.yaml

```diff
@@ -1,3 +1,5 @@
+# yaml-language-server: $schema=https://sap.github.io/ui5-tooling/schema/ui5.yaml.json
+
 specVersion: "2.5"
 metadata:
```

Comment:
I know it is not exactly relevant for this PR but why do we have that file here? This file is defined in th`ui5-application-writer` so why do we redefine it here?

Reply:
@ullasholla I can answer my own question. The files is obsolete, I have removed it in 48d6fe7  and all snapshot tests run through as before.

Reply:
Thanks. Going to mark this as resolved.